router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -219,7 +246,7 @@ func (b *BingAdsBulkUploader) Upload(destination *backendconfig.DestinationT, as
 	// parameters.PollUrl = pollUrl
 	importParameters, err := json.Marshal(parameters)
 	if err != nil {
-		panic("Errored in Marshalling" + err.Error())
+		b.logger.Error("Errored in Marshalling parameters" + err.Error())
```

Comment:
currently we are ignoring the error after logging error, should we return error so that caller can handle it accordingly?

Reply:
@ItsSudip